roles/tests/test_api.py

```diff
@@ -20,7 +23,8 @@ class TestRoleApi(LoreTestCase):
     roles_init_new_repo is tested indirectly from
     the repository model where it is called
     """
-    # pylint: disable=too-many-instance-attributes
+    # there are too many self assignment in the init: disabling the check
+    # pylint: disable=too-many-instance-attributes, invalid-name
```

Comment:
If I move it to the init, it does not disable the pylint check: I guess it refers to the class and not to the init